crates/common/src/raindex_client/vaults.rs

```diff
@@ -276,36 +288,35 @@ impl RaindexVault {
     /// const calldata = result.value;
     /// // Do something with the calldata
     /// ```
-    #[wasm_export(js_name = "getDepositCalldata", unchecked_return_type = "Hex")]
+    #[wasm_export(
+        js_name = "getDepositCalldata",
+        return_description = "Encoded transaction calldata as hex string",
+        unchecked_return_type = "Hex"
+    )]
     pub async fn get_deposit_calldata(
         &self,
-        deposit_amount: String,
+        #[wasm_export(
+            param_description = "Amount to deposit in token's smallest unit (e.g., \"1000000000000000000\" for 1 token with 18 decimals)"
+        )]
+        amount: String,
```

Comment:
why can't this be the user facing amount? e.g. just 1 instead of 1e18. we already know the decimals for the token.